bin/rucio

```diff
@@ -1403,10 +1404,14 @@ def update_rule(args):
     if args.lifetime:
         options['lifetime'] = None if args.lifetime.lower() == "none" else int(args.lifetime)
     if args.locked:
-        if args.locked == "True":
+        if args.locked.title() == "True":
             options['locked'] = True
-        elif args.locked == "False":
+        elif args.locked.title() == "False":
             options['locked'] = False
+        else:
+            logger.error('Locked must be True or False')
+            return FAILURE
+        options['locked'] = True
```

Comment:
It will work if you delete this line. This line would override everything with `True`.

Reply:
Yea, my bad :( ...Removed it now